src/Controls/tests/Core.UnitTests/TypedBindingUnitTests.cs

```diff
@@ -864,21 +896,27 @@ public object ConvertBack(object value, Type targetType, object parameter, Cultu
 		}
 
 #if !WINDOWS_PHONE
```

Comment:
I assume this is always true. Should we remove the `#if`?

Reply:
Yes, this is old from Xamarin.Forms, so you can remove it if you're touching this file. 👍